buildbot/master/files/config/master.cfg

```diff
@@ -232,7 +232,7 @@ arm64_factory = create_servo_factory([
     steps.Compile(command=["./mach", "build", "--rel", "--target=aarch64-unknown-linux-gnu"], env=arm64_compile_env),
 ])
 
-windows_compile_env = dict({'PATH': 'C:\\msys64\\mingw64\\bin;C:\\msys64\\usr\\bin\\C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\Amazon\\cfn-bootstrap\\',
+windows_compile_env = dict({'PATH': r'C:\msys64\mingw64\bin;C:\msys64\usr\bin\C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Amazon\cfn-bootstrap',
```

Comment:
Yes, much easier to work with. (For some reason GitHub highlights these weird, though.)

Did you remove the slash at the end on purpose?